.github/workflows/main.yaml

```
@@ -165,7 +169,8 @@ jobs:
             --cov=pytest_pyodide \
             --dist-dir=./pyodide-dist/ \
             --runner=${{ inputs.runner }} \
-            --rt ${{ inputs.browser }}
+            --rt ${{ inputs.browser }} \
```

```suggestion
            --rt=${{ inputs.browser }} \
```
Without this, `pytest-extra-args` will be push to runtime list

---

Oh, okay this change breaks [one test](https://github.com/pyodide/pytest-pyodide/blob/a9db5589eb0b0986ade68833ec046374288e3f58/.github/workflows/filtermatrix.py#L29-L33), which passes multiple browsers to inputs.browser. I think I added that to test whether `--rt` correctly handles multiple parameters. But I think we can remove that test from filtermatrix.py, regarding that users who use this action don't need that test.